Makefile

```diff
@@ -0,0 +1,754 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
@icecube45 plz remove

Reply:
Argh I thought I had caught everything from when I accidently cmaked in the base directory.